docs/inbound-agent.md

```diff
@@ -33,7 +33,12 @@ This mechanism is not recommended, is deprecated, and may not be supported or av
 ### Download JNLP file
 Another mechanism, shown in the above status page fragment, runs the agent from a script or command-line to retrieve the JNLP file. 
 This mechanism does not use JNLP or WebStart but uses the downloaded file to obtain connection information. 
-Internally, this is a two-step process: 1) connect to an HTTP(S) port to retrieve the connection information, and 2) extract that information and connect to the TCP port.
+
+For an inbound agent using the TCP port, this is a two-step process:
+* Connect to an HTTP(S) port to retrieve the connection information.
+* Extract that information and connect to the TCP port.
+
+(When using the `-webSocket` option, only a single connection needs to be made.)
```

Comment:
Also for Direct inbound TCP agent connection in #338. Would be great to reference it as well

Reply:
Maybe…seems redundant. We might consider rewriting the whole page to discuss the various modes more clearly. For example, the Docker image bypasses the JNLP entry point and uses a different entry point in the JAR; this is alluded to in the direct connection section but there is no mention that this can be done even when _not_ using direct connection mode.

Reply:
I tossed this page together to get a minimal amount of information out there, though describing the minimum still took quite a bit. For a long time I've been wanting to get better user-facing documentation on how to use agents and distribute Jenkins. It's still on a list of things to do and we talk about it from time to time.

For now, let's leave this page as currently proposed in this PR.